lib/wpc/slip.c

```diff
@@ -125,35 +118,35 @@ static uint32_t slip_decode_buffer(uint8_t* buffer,
  *          buffer of the escaped frame
  * \param   len_buffer
  *          length of the buffer to store the escaped frame
- * \return  length of the escaped frame or -1 if the escaped frame cannot feet the provided buffer
+ * \return  length of the escaped frame or -1 if the escaped frame cannot feet
+ * the provided buffer
  */
-static uint32_t slip_encode_buffer(uint8_t * buffer,
-                                   uint32_t len,
-                                   uint8_t * buffer_escaped,
-                                   uint32_t len_escaped)
+static uint32_t
+slip_encode_buffer(uint8_t * buffer, uint32_t len, uint8_t * buffer_escaped, uint32_t len_escaped)
 {
     uint8_t write = 0;
     uint32_t read;
-    for (read = 0; read < len; read++) {
+    for(read = 0; read < len; read++)
+    {
         uint8_t current = buffer[read];
-        if (current == END_SLIP_OCTET || current == ESC_SLIP_OCTET) {
-            //Check for buffer_escaped overflow
-            if (write >= len_escaped)
+        if(current == END_SLIP_OCTET || current == ESC_SLIP_OCTET)
+        {
+            // Check for buffer_escaped overflow
+            if(write >= len_escaped)
                 return WPC_INT_WRONG_BUFFER_SIZE;
             buffer_escaped[write++] = ESC_SLIP_OCTET;
             current = (current == END_SLIP_OCTET ? END_SUBS_OCTET : ESC_SUBS_OCTET);
         }
-        //Check for buffer_escaped overflow
-        if (write >= len_escaped)
+        // Check for buffer_escaped overflow
+        if(write >= len_escaped)
             return WPC_INT_WRONG_BUFFER_SIZE;
 
         buffer_escaped[write++] = current;
     }
     return write;
 }
 
-int Slip_decode(uint8_t * buffer,
-                uint32_t len)
+int Slip_decode(uint8_t * buffer, uint32_t len)
 {
     int decoded_len;
     uint16_t crc, crc_from_frame;
```

Comment:
Should notice two variable definitions and one type. Should be:

uint16_t crc;
uint16_t crc_from_frame;